docs/source/tutorials/filtsmooth/linear_bayesian_filtering_smoothing.ipynb

```diff
@@ -0,0 +1,4552 @@
+{
```

Comment:
<p>the align block somehow does not render well -- at least in ReviewNB.</p><p> </p><p>also: "Note that this can be generalized to a linear, time-varying state space model" </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probabilistic-numerics/probnum/pull/292/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>